config.toml

```diff
@@ -37,7 +37,7 @@ theme = 'hive'
     precommit = "https://cwiki.apache.org/confluence/display/Hive/Hive+PreCommit+Patch+Testing"
     faq = "https://cwiki.apache.org/confluence/display/Hive/HiveDeveloperFAQ"
     vcs = "/development/versioncontrol/"
-    commiter = "https://cwiki.apache.org/confluence/display/Hive/BecomingACommitter"
+    commiter = "/community/becomingcommitter/"
```

Comment:
commiter? Is it a typo?

Reply:
It's not something that is visible to end-users but I fixed in https://github.com/apache/hive-site/pull/27/commits/261d314b4ab454877509eb96491a5d188b1e2d61